lib/sidekiq_adhoc_job.rb

```diff
@@ -34,6 +34,14 @@ def self.init
     Sidekiq::Web.register(SidekiqAdhocJob::Web)
     Sidekiq::Web.tabs['adhoc_jobs'] = 'adhoc-jobs'
     Sidekiq::Web.locales << File.expand_path('sidekiq_adhoc_job/web/locales', __dir__)
+
+    assets_path = File.expand_path('sidekiq_adhoc_job/web/assets', __dir__)
+
+    Sidekiq::Web.use Rack::Static, urls: ['/javascript'],
+                               root: assets_path,
+                               cascade: true,
+                               header_rules: [[:all, { 'cache-control' => 'private, max-age=86400' }]]
```

Comment:
very nit haha, usually we have rubocop linter so this probably will be picked up by that, but I think this repo maybe doesn't use the linter

```suggestion
    Sidekiq::Web.use Rack::Static, urls: ['/javascript'],
                                   root: assets_path,
                                   cascade: true,
                                   header_rules: [[:all, { 'cache-control' => 'private, max-age=86400' }]]
```